### PR TITLE
Discuss existing proposals

### PR DESCRIPTION
**Pull request by felixlinker**

### CODE REVIEW

**Comment by felixlinker** [comment]
@mstojens Would be great if you could review!

File: draft-linker-digital-emblem.md [hunk 1]
```diff
@@ -212,6 +230,23 @@ However, marking entire networks should drastically ease the burden of deploymen
...
+We focus here on ADEM as it was selected by the ICRC as the most suitable proposal for a digital emblem.
+
+ADEM was designed following the requirements laid out in this draft.
+ADEM follows a decentralized trust model and utilizes existing infrastructure wherever possible, e.g., it provides accountability through the Certificate Transparency infrastructure {{?RFC6962}}.
+ADEM was designed to be distributed over many channels and as such is not bound to one mode of transportation.
+The academic paper foresees distribution of digital emblems via TLS, DNS, and UDP.
+Next to its prototype, ADEM's security was thoroughly evaluated and formal proofs of security are described in the academic publication.
+
+Once there is consensus on the scope of a digital emblem, the plan is to propose ADEM as a basis for designing a digital emblem.
+Although ADEM was evaluated on many occasions, it is subject to change.
```
Comment (tojens-ietf) [inline draft-linker-digital-emblem.md]:
```suggestion
Although ADEM has gone through several iterations in collaboration with the ICRC, it is expected that it will be refined by the IETF WG that adopts it to ensure industry interoperability with good protocol practices.
```
I'm not insisting on this literal text change word for word, but I hope it illustrates the point I'm suggesting (being direct about what I know to be true, which is "this is no rough draft, but we're also open to change as part of the multistakeholder standards development process"

File: draft-linker-digital-emblem.md [hunk 2]
```diff
@@ -212,6 +230,23 @@ However, marking entire networks should drastically ease the burden of deploymen
 Notably, a digital emblem cannot be applied to infrastructure that is used for both services worthy and unworthy of protection.
 A digital emblem must always identify assets that only serve purposes that enjoy protection under IHL.
 
+# Proposed Designs for a Digital Emblem {#proposals}
+
+Two designs were proposed to solve the problem of a digital emblem.
+First, JHU APL proposed a DNS-based solution for a digital emblem {{?I-D.haberman-digital-emblem}}.
+Second, CECYT (a joint endeavour between ETH Zurich and Bonn University) proposed *An Authentic Digital EMblem (ADEM)*, which was initially described in an academic publication {{ADEM}}, but was also specified technically {{ADEM-SPEC}} and has openly accessible, working prototypes ([Go library and CLI](https://github.com/adem-wg/adem-proto), [Browser extension](https://github.com/adem-wg/adem-chrome)).
+We focus here on ADEM as it was selected by the ICRC as the most suitable proposal for a digital emblem.
```
Comment (tojens-ietf) [inline draft-linker-digital-emblem.md]:
```suggestion
We focus here on ADEM as it was selected by the ICRC as the most suitable proposal for their digital emblem requirements. In terms of scope, this means meeting the need to apply an emblem to digital assets without a need to apply digital emblems to physical assets, which goes beyond the ICRC's needs.
```
I think something like this (doesn't have to be exactly this) would help put context around the decision and not just say "ADEM is the winner, moving on" which the original comes across as to a reader with zero background in the digital emblem space. It's not that JHU APL's solution can't solve other problems, but that ADEM focuses on the problem space the ICRC cares about.

**Review by tojens-ietf — CHANGES_REQUESTED** [review]

File: draft-linker-digital-emblem.md (hunk 1 above)
Comment (felixlinker) [reply draft-linker-digital-emblem.md]: Reads much better than my version and is exactly what I could not but wanted to express! :)

File: draft-linker-digital-emblem.md (hunk 2 above)
Comment (felixlinker) [reply draft-linker-digital-emblem.md]: I think we should fight for the term "digital emblem" meaning the digital emblem as discussed in our draft. You might use other technology to solve other problems, but we should not call these other problems digital emblems. If you want to replace the physical emblems with something digital, you must go through the ICRC, and they fight this idea.

Happy to tone down what ADEM was selected for, but I think we should really insist on "'digital emblem' applied to physical assets" ≠ "digital emblem."

Comment (felixlinker) [reply draft-linker-digital-emblem.md]: Maybe we can go another way and rather stress that we understand "digital emblem" to only apply to digital assets and then list the reasons for the ICRC why they want that. This would also help to kickstart the discussion on this topic at the IETF. (And reduce surprises to ADs!)

Comment (tojens-ietf) [reply draft-linker-digital-emblem.md]: Ah, I see the nuance. Let me try again.

Comment (tojens-ietf) [inline draft-linker-digital-emblem.md]:
```suggestion
We focus here on ADEM as it was selected by the ICRC as the most suitable proposal for a digital emblem given its scope of applying emblems to digital assets and using the existing physical emblems for labeling physical assets.
```

**Comment by felixlinker** [comment]
Thanks for the suggestions, @mstojens ! I edited your third, updated suggestions slightly (mostly dropping some words). Good to merge? I think so!

**Comment by tojens-ietf** [comment]
LGTM!
